apis/fluentbit/v1alpha2/clusterfilter_types.go

```diff
@@ -40,7 +40,7 @@ type FilterSpec struct {
 	// Use this option if you want to use the full regex syntax.
 	MatchRegex string `json:"matchRegex,omitempty"`
 	// A set of filter plugins in order.
-	FilterItems []FilterItem `json:"filters,omitempty"`
+	FilterItems []FilterItem `json:"clusterFilters,omitempty"`
```

Comment:
We needn't change fields except CRD name to cluster like below to make the migration from namespaced CR to cluster CR easier

filters -> clusterFilters

Reply:
fixed